src/views/projects/SourceBrowser/FileDiff.svelte

```diff
@@ -16,6 +17,59 @@
   export let projectId: string;
 
   let collapsed = false;
+  let selection: Selection | undefined = undefined;
+
+  onMount(() => {
+    window.addEventListener("click", deselectHandler);
+    window.addEventListener("hashchange", updateSelection);
+  });
+
+  onDestroy(() => {
+    window.removeEventListener("click", deselectHandler);
+    window.removeEventListener("hashchange", updateSelection);
+  });
+
+  onMount(() => {
```

Comment:
Can we combine these into one `onMount`?

Reply:
we can, but I find separate `onMount` and `onDestroy` more readable, your call.

Reply:
Yeah, let's do one `onMount` to be consistent with the rest of the codebase.

Reply:
I don't follow, why would we want to change something there?

Reply:
gotcha, https://github.com/radicle-dev/radicle-interface/pull/942/commits/5ad8e2ef11a983ed3d22bd9e23c5093ed8f6820a